### ISSUE TITLE
Incorrect License used when creating container

### ISSUE THREAD

**Issue by nickrobbo** [NAVBaaS/GitIntegration#2]
When setting up the extension I specified a license to use, which is compatible with the NAV version I intend to use.  When creating a container I see that a different license is used?
```

Starting Internet Information Server
Modifying Service Tier Config File with Instance Specific Settings
Starting NAV Service Tier
Using license file 'c:\run\my\license.flf'
Import License
```
It seems my specified license is not used, then I get an error further along in the process when the extension tries to import the objects?
```

VERBOSE: Importing objects.
Import-NAVApplicationObject : : [18023808] You do not have permission to insert into the VAT Entry table.
[0] The import stopped at line 64329.
At C:\Users\robertsn\.vscode\extensions\navbaas.git-1.0.0\ps\m\NavHelper.psm1:249 char:9
+         Import-NAVApplicationObject `
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Import-NAVApplicationObject
```

I'm not sure if this is because we do not have a full Partners license? Or if my license is not used correctly.

**Comment by richardrobberse** [comment]
When creating the container (it uses the navcontainerhelper in the background) it copies the license to the container, and 'c:\run\my\license.flf' is the path where it is copied to.
As far as I can see it uses the correct license, the error is most likely caused by the fact that you don't have a full partner license and you're not allowed to import text files?

Please check which license is used in C/SIDE by going to 'Tools' -> 'License Information'

**Comment by nickrobbo** [comment]
yeah I have just checked, it is due to us not having full partner license.  We have a license which allows us to modify objects, but some standard objects we cannot touch, such as this table.

This is a shame, as it seems this the only thing stopping us using the extension, back to the drawing board.  It seems that even if I specify -ExportTxtSkipUnlicensed to my script which gets the objects in the first place, they are not skipped.

**Comment by richardrobberse** [comment]
Hm, that's a shame.
I can't really help you with that one I guess... :(

**Comment by nickrobbo** [comment]
I think the only way around it for us, is to find out which objects we have truly modified and make sure the "Version List" field reflects this, then only export those objects from the start.  It isn't the nicest, but with our license limitations it might be the only way.  Issue is then when a partner modifies one of those objects for us :( .

Thanks for your help anyway.

**Comment by richardrobberse** [comment]
You're welcome.
I would encourage you to only track files in source control that are modified.
Otherwise, you're keeping track of standard objects as well and there's no need to do since you can spin up a new standard container at any time.

**Comment by nickrobbo** [comment]
It has just occurred to me that we could import our base objects via a fob, could you add some functionality to do this? As this would mean we can import the base objects we do not have access to initially as it skips these license checks.  There might be other users in the same situation as us, so it might be helpful for more users.

**Comment by richardrobberse** [comment]
My goal is to keep this extension as simple as possible and not overcomplicate it by adding commands for all the corner cases that might occur at some point.
Importing the fob is perfectly possible through C/SIDE, right?

In the next update I will provide commands to:
- Preview the synchronization of the container without performing the actual import, export etc.
- Sync schema changes - because this doesn't work through C/SIDE
- Compile objects - compiles objects inside the container because some dependencies might not be available on the host.

**Comment by nickrobbo** [comment]
right I see, that is perfectly fine, I understand.  So you think we should initialize the container, then import the objects in via C\Side, leaving the modified folder empty? Then just continue with the normal workflow?

**Comment by richardrobberse** [comment]
Yes, I think so but I'm not sure what your development process looks like.
Is it the case that you develop additional customization on top of an add-on or something?

**Comment by nickrobbo** [comment]
No, I work at an end user, and we do all our modifications ourselves.  We have only one Add-In (Document Capture).  The issue is there are a few tables outside of our license range which were modified in the past by a partner (things relating to Margin VAT etc).  So we have inherited those.  Right now we work in sprints, in quite a small team and I am just trying to get us into using SCM and TDD.

**Comment by richardrobberse** [comment]
I see, cool! I'm really glad you're using my extension already  :)
So you will be able to export the objects that you work on as text?

P.S. I've just fixed your NuGet issue but I have some more work to do before I release the new version.

**Comment by nickrobbo** [comment]
Yes I can export/import objects.  I have created a few scripts to automate our current deployment process which looks to Jira to find the latest issues requiring release etc.  I have exported all our objects and split them, in order to build the base app.  This is where I run into this issue importing the objects.  It seems that the switch -ExportTxtSkipUnlicensed isn't as clever as I thought it was.
